src/math/lp/nla_types.h

```diff
@@ -97,6 +97,7 @@ namespace nla {
         new_lemma& operator&=(const factorization& f);
         new_lemma& operator&=(lpvar j);
         new_lemma& operator|=(ineq const& i);
+        new_lemma& operator&=(ineq const& ineq);
```

Comment:
use of & is likely not following the conventions.
& is for assumptions, |= is for conclusions.
A1 & A2 & A3 => C1 | C2 | C3